### DISCUSSION TITLE
is it possible to change lyrics font?

### DISCUSSION THREAD

**Discussion by CuiYongen** [TT-ReBORN/Georgia-ReBORN#68] [Q&A] [answered]
is it possible to change lyrics font? can't find settings in lyrics show 3

**Comment by TT-ReBORN** [comment, marked as answer]
Yes it possible, open georgia-reborn-main.js and go to here:

https://github.com/TT-ReBORN/Georgia-ReBORN/blob/1a817765459fde6f3430bea0e05ceb24c14a911e/profile/georgia-reborn/js/georgia-reborn-main.js#L58

and add a new var, e.g:
const fontLyrics = 'the new font name comes here'; // your new font for Lyrics

then go to:
https://github.com/TT-ReBORN/Georgia-ReBORN/blob/1a817765459fde6f3430bea0e05ceb24c14a911e/profile/georgia-reborn/js/georgia-reborn-main.js#L164

and change
`ft.lyrics = font(fontRegular, pref.lyricsFontSize || 20, 1);`
to 	
`ft.lyrics = font(fontLyrics, pref.lyricsFontSize || 20, 1);`

-TT
